lib/apiservers/engine/backends/container.go

```diff
@@ -755,7 +755,7 @@ func (c *Container) imageExist(imageID string) (storeName string, err error) {
 	return host, nil
 }
 
-func (c *Container) getImageMetadataFromStoragePL(image string) (*viccontainer.VicContainer, error) {
+func (c *Container) getImageMetadataFromImageC(image string) (*viccontainer.VicContainer, error) {
```

Comment:
That was just a function name change.  Create still uses that.  This PR wasn't really meant to touch everything that uses image metadata.  I'll leave the changes for Create to @caglar10ur.  I think he's slated for that.